src/components/Book.js

```diff
@@ -1,39 +1,76 @@
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { RemoveBook, FetchBooks } from '../redux/books/booksSlice';
+import styles from './Book.module.css';
 
-const Book = ({ bookProp, id }) => {
+function Book({ bookProp, id }) {
   const dispatch = useDispatch();
+  const randomPercentage = Math.floor(Math.random() * 100) + 1;
+  const randomNumber = Math.floor(Math.random() * 10) + 1;
   return (
-    <>
-      <div className="book-details">
-        <h4 className="genre">fiction</h4>
-        <input type="text" className="title" value={bookProp[0].title} readOnly />
-        <h4 className="author">{bookProp[0].author}</h4>
+    <div className={styles.book}>
+      <div className={styles['book-details']}>
+        <div className={styles.maininfo}>
+          <h4 className={styles.genre}>fiction</h4>
+          <input
+            type="text"
+            className={styles.title}
+            value={bookProp[0].title}
+            readOnly
+          />
+          <h4 className={styles.author}>{bookProp[0].author}</h4>
+          <div className={styles.actions}>
+            <ul className={styles.actionsul}>
```

Comment:
- [ ] Remove spaces here in order to match better the design 
![image](https://github.com/yashodhicy/BookStore/assets/62705947/6f8fdb64-a072-4498-8acd-9248fdd4b116)